### ISSUE TITLE
ScaleAnimation does not work together with AngleAnimation

### ISSUE THREAD

**Issue by steffimueller** [IFTTT/JazzHands#1678]
I tried to use ScaleAnimation together with AngleAnimation these two does not work together.

**Comment by jhersh** [comment]
Sure they do. Make sure you're using two animations, and check out how the [wordmark in the example app](https://github.com/IFTTT/JazzHands/blob/master/Example/JazzHandsDemo/IFTTTJazzHandsViewController.m#L127) both rotates and translates its frame at the same time.

**Comment by steffimueller** [comment]
This is not the same in your example when using wordmark you use a 
```
IFTTTFrameAnimation 
IFTTTAngleAnimation
```
What I was trying to do is: 
```
IFTTTScaleAnimation 
IFTTTAngleAnimation
```
These two do not work together. When you use the frame animation for scaling I think your image rendering is much worse then when using a IFTTTScaleAnimation (scale transformation).

**Comment by lauraskelton** [comment]
@steffimueller You're right, they don't currently work together, since they both overwrite the same view.transform property. We have a fix in the works!

**Comment by confile** [comment]
@lauraskelton Could you please push this fix.

**Comment by lauraskelton** [comment]
@steffimueller @confile I just released JazzHands 2.0, which allows `IFTTTScaleAnimation`, `IFTTTRotationAnimation` (which replaces `IFTTTAngleAnimation` and uses degrees instead of radians), and `IFTTTTranslationAnimation` to operate on the same view without conflicting.

I also added a unit test that makes sure for the future that all of our animations can work on the same view at the same time.

**Comment by confile** [comment]
@lauraskelton Awesome. Thank you!
